### DISCUSSION TITLE
Comments are broken on my blog

### DISCUSSION THREAD

**Discussion by CranberryPie1** [posativ/isso#775] [Q&A]
I figured out the issue with the admin panel I updated my old post on not being able to access the admin panel with a fix.

On to comments not working on the blog, I got complaints from 2 users that weren't able to post nor reply on my blog. I checked the error logs on both nginx and isso and found nothing.

I decided to do a little test and found the following. User **Test#1** and **Test#2**. Use Test#2 can't reply to user Test#, different browser and used a VPN. Here comes the fun part, If I use Test#2 to reply to Test#1 and decide to post a individual comment with Test#2, I can't post anything!

![test-1](https://user-images.githubusercontent.com/69950364/91272880-71bc8d80-e74a-11ea-94a8-450617bf3eae.png)
![Test-2](https://user-images.githubusercontent.com/69950364/91272888-73865100-e74a-11ea-8166-215df28ad2aa.png)
```

                <script data-isso="https://isso.domain.com/"
                        data-isso-gravatar="true"
                        data-isso-avatar="false"
                        data-isso-reply-notifications="true"
                        data-isso-vote="false"
                        data-isso-require-author="true"
                        data-isso-lang="{{@site.lang}}"
                    src="https://isso.domain.com/js/embed.min.js"></script>
                <section id="isso-thread"></section>
```

**Comment by jefft** [comment]
Try turning off gravatars (set `data-isso-gravatar="false"` or remove that attribute completely), and fix the `data-isso-lang`. You could simplify it down to:
```javascript
<script type="text/javascript" 
async
 src="https://isso.domain.com/js/embed.min.js"
 data-isso="https://isso.domain.com/"
 data-isso-reply-notifications="true">
</script>
<section id="isso-thread"></section>
```

**Comment by CranberryPie1** [comment]
I have gravatar enabled and avatar removed from the config and it shows up as in the image you see. Why when adding avatar=false to the config does it cause gravatar to break?

Removing lang seemed to have partially fix replying to others. My next bet is possibly a permission setting.

![test1](https://user-images.githubusercontent.com/69950364/91350026-1cb56180-e7b4-11ea-942a-c3883a0cd2b8.png)

**Comment by jefft** [comment]
Perhaps you weren't reloading isso after disabling gravatars? If you get a broken image that means gravatars are enabled:

https://github.com/posativ/isso/blob/3d0fdffcb70bcff3c7f7ae28285e918a06655998/isso/js/app/text/comment.jade#L4

If you disable gravatars, you will instead get a SVG image:

![image](https://user-images.githubusercontent.com/205995/91368824-64d49280-e84d-11ea-827a-b6f9cd26ac86.png)

I tried turning on gravatars on my site and nothing happened. Perhaps the support is broken.

**Comment by CranberryPie1** [comment]
I restarted Isso. I fixed the permission settings (which sort of fixed the display/reply issue.) 

It could either be support for gravatars is broken or Nginx.

**Comment by ix5** [comment]
Would you mind checking if the suggested solutions to gravatars not appearing in https://github.com/posativ/isso/issues/729 might help?

I'm afraid without a full config file and (current) client side settings as well as a link to your (broken) comment section on your blog, it might be difficult to help you.

**Comment by ix5** [comment]
@CranberryPie1 could you kindly verify this is still an issue? Without your cooperation, there is no chance to solve this issue.
